examples/polypharmacy_gripnet/config.py

```diff
@@ -1,42 +1,56 @@
+from typing import List, Tuple
+
 from yacs.config import CfgNode
 
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 # Config definition
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 
 C = CfgNode()
 
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 # Dataset
-# -----------------------------------------------------------------------------
+# ---------------------------------------------------------
 C.DATASET = CfgNode()
 C.DATASET.ROOT = "./data"
-C.DATASET.NAME = "PoSE"
-C.DATASET.URL = "https://github.com/pykale/data/raw/main/graphs/pose.pt"
+C.DATASET.NAME = "pose"
+C.DATASET.URL = "https://github.com/pykale/data/raw/main/graphs/pose_pyg_2.pt"
 
-# ---------------------------------------------------------------------------- #
+# ---------------------------------------------------------
 # Solver
-# ---------------------------------------------------------------------------- #
+# ---------------------------------------------------------
 C.SOLVER = CfgNode()
-C.SOLVER.SEED = 2020
+C.SOLVER.SEED = 1111
 C.SOLVER.BASE_LR = 0.01
-C.SOLVER.LR_MILESTONES = [30, 60, 90]
-C.SOLVER.LR_GAMMA = 0.1
-C.SOLVER.MAX_EPOCHS = 5
-C.SOLVER.WARMUP = False
-C.SOLVER.WARMUP_EPOCHS = 100
-
-# ---------------------------------------------------------------------------- #
-# GripNet configs
-# ---------------------------------------------------------------------------- #
-C.GRIPN = CfgNode()
-C.GRIPN.GG_LAYERS = [32, 16, 16]
-C.GRIPN.GD_LAYERS = [16, 32]
-C.GRIPN.DD_LAYERS = [sum(C.GRIPN.GD_LAYERS), 16]
-
-# ---------------------------------------------------------------------------- #
+C.SOLVER.MAX_EPOCHS = 66
+C.SOLVER.LOG_EVERY_N_STEPS = 1
+
+# ---------------------------------------------------------
+# GripNet supervertex configs
+# ---------------------------------------------------------
+# create configuration for each supervertex
```

Comment:
If you move the comments to next to the config parameter, then this line is not needed and can be removed. It does not add much addtion to line 29. You can edit 29 if you like to be clearer.